azure-boards-common/build.gradle

```diff
@@ -1,9 +1,13 @@
 ext.moduleName = 'com.synopsys.integration.azure.boards.common'
 
 dependencies {
-    implementation 'com.google.cloud:libraries-bom:2.2.0'
-    api 'com.google.oauth-client:google-oauth-client'
-    api 'com.google.api-client:google-api-client:1.30.9'
-    api 'com.google.http-client:google-http-client-apache-v2:1.36.0'
-    implementation 'com.synopsys.integration:integration-common:22.2.0'
+    implementation platform(project(':alert-platform'))
+
+    implementation 'com.google.cloud:libraries-bom'
+    implementation 'com.google.oauth-client:google-oauth-client'
```

Comment:
I didn't see this in the platform project explicitly declared.  Is this a transitive of another API project?

Reply:
It comes from spring I think.

Reply:
Probably a transitive of int-jira-common most likely.

Reply:
It is indeed.